skyportal/handlers/api/enum_types.py

```diff
@@ -0,0 +1,60 @@
+from baselayer.app.access import auth_or_token
+
+from ..base import BaseHandler
+from ...enum_types import (
+    ALLOWED_SPECTRUM_TYPES,
+    ALLOWED_MAGSYSTEMS,
+    ALLOWED_BANDPASSES,
+    THUMBNAIL_TYPES,
+    FOLLOWUP_PRIORITIES,
+    ALLOWED_API_CLASSNAMES,
+)
+
+
+class EnumTypesHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        """
+        ---
+        description: Retrieve enum types in the DB
+        tags:
+          - system_info
+        responses:
+          200:
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Success'
+                    - type: object
+                      properties:
+                        data:
+                          type: object
+                          properties:
+                            ALLOWED_SPECTRUM_TYPES:
+                              type: array
+                              description: list of allowed spectrum types
+                            ALLOWED_MAGSYSTEMS:
+                              type: array
+                              description: list of allowed magnitude systems
+                            ALLOWED_BANDPASSES:
+                              type: array
+                              description: list of allowed bandpasses
+                            THUMBNAIL_TYPES:
+                              type: array
+                              description: list of allowed thumbnail types
+                            FOLLOWUP_PRIORITIES:
+                              type: array
+                              description: list of allowed followup priorities
+                            ALLOWED_API_CLASSNAMES:
+                              type: array
+                              description: list of allowed API classnames
+        """
+        data = {}
+        data["ALLOWED_SPECTRUM_TYPES"] = sorted(ALLOWED_SPECTRUM_TYPES)
```

Comment:
Why are we sorting these? I think in some cases the original order of the enums matters. For example the spectrum types order determines the order of the tabs on the plot.

Reply:
Just because then finding them in the dropdown list isnt so bad. I can undo it.

Reply:
That's a good reason. Maybe sorting some of them? Or sorting on the front end just to make the dropdowns easier to read for specific enums like the bandpasses.

Reply:
Yeah sorting on the front-end made sense, I did that.